glue/viewers/profile/layer_artist.py

```diff
@@ -147,16 +148,17 @@ def _update_visual_attributes(self):
 
         self.redraw()
 
-    def _update_profile(self, force=False, **kwargs):
+    def _update_profile(self, force=True, **kwargs):
```

Comment:
Is this intentional? It is a big change.

Reply:
Yeah, it is intentional. We would like the sliders to be able to update the 1D profiles automatically. This seems to be the most natural place/way to do so.